webapp/channels/src/utils/license_utils.ts

```diff
@@ -84,10 +84,6 @@ export function getIsGovSku(license: ClientLicense) {
     return license?.IsGovSku === 'true';
 }
 
-export function isEnterpriseOrE20License(license: ClientLicense) {
-    return license?.SkuShortName === LicenseSkus.Enterprise || license?.SkuShortName === LicenseSkus.E20;
-}
-
 export const isEnterpriseLicense = (license?: ClientLicense) => {
```

Comment:
Do we want to expand to include the new sku funcion check?

Reply:
I'll address this when I add the webapp-side changes to this PR. So far this is mostly server only .

Reply:
Webapp changes are done as well.